specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/blob.json

```diff
@@ -1042,7 +1051,11 @@
         },
         "allowProtectedAppendWrites": {
           "type": "boolean",
-          "description": "This property can only be changed for unlocked time-based retention policies. When enabled, new blocks can be written to an append blob while maintaining immutability protection and compliance. Only new blocks can be added and any existing blocks cannot be modified or deleted. This property cannot be changed with ExtendImmutabilityPolicy API"
+          "description": "This property can only be changed for unlocked time-based retention policies. When enabled, new blocks can be written to an append blob while maintaining immutability protection and compliance. Only new blocks can be added and any existing blocks cannot be modified or deleted. This property cannot be changed with ExtendImmutabilityPolicy API."
+        },
+        "allowProtectedAppendWritesAll": {
+          "type": "boolean",
+          "description": "This property can only be changed for unlocked time-based retention policies. When enabled, new blocks can be written to both 'Appened and Bock Blobs' while maintaining immutability protection and compliance. Only new blocks can be added and any existing blocks cannot be modified or deleted. This property cannot be changed with ExtendImmutabilityPolicy API. The 'allowProtectedAppendWrites' and 'allowProtectedAppendWritesAll' properties are mutually exclusive."
```

Comment:
(Optional ) We should change the ` "name": "parameters",` to  "name": "immutabilityPolicy", on line 532 and 750 in blob.json. 

If not change, the generated SDK will has the parameter name for input immutabilityPolicy as "parameters", which doesn't make sense.
Before add "allowProtectedAppendWritesAll", this issue not appear in SDK, since immutabilityPolicy  only has 2 properties can be input, the Set/extend immutabilityPolicy  API will have the 2 immutabilityPolicy  properties just as the API paramters of the API; but after add one more property "allowProtectedAppendWritesAll", the Set/extend immutabilityPolicy  API will have the whole immutabilityPolicy  has parameter. 

The .net SDK API with current swagger (See "ImmutabilityPolicy parameters = default(ImmutabilityPolicy) ")

```
public static ImmutabilityPolicy CreateOrUpdateImmutabilityPolicy(this IBlobContainersOperations operations, string resourceGroupName, string accountName, string containerName, ImmutabilityPolicy parameters = default(ImmutabilityPolicy), string ifMatch = default(string))
```

The API after change  ` "name": "parameters",` to  "name": "immutabilityPolicy" in swagger: (See "ImmutabilityPolicy immutabilityPolicy = default(ImmutabilityPolicy)")
```
public static ImmutabilityPolicy CreateOrUpdateImmutabilityPolicy(this IBlobContainersOperations operations, string resourceGroupName, string accountName, string containerName, ImmutabilityPolicy immutabilityPolicy = default(ImmutabilityPolicy), string ifMatch = default(string))
```